packages/color-block/tailwind.config.js

```diff
@@ -0,0 +1,27 @@
+module.exports = {
+    presets: [require('@frontify/fondue/tailwind')],
+    content: ['src/**/*.{ts,tsx}'],
+    corePlugins: {
+        preflight: false,
+    },
+    theme: {
+        extend: {
+            width: {
+                'list-color-types': 'calc(100% - 306px)',
+                'cards-card': 'calc(25% - 16px)',
+            },
+            boxShadow: {
+                inset: 'inset 0 0 0 1px rgba(8, 8, 8, 0.1)',
+                'inset-top': 'inset 0 1px 0 0 rgba(8, 8, 8, 0.1)',
+                'inset-bottom': 'inset 0 -1px 0 0 rgba(8, 8, 8, 0.1)',
+                'inset-full': 'inset 0 0 0 1px rgba(8, 8, 8, 0.1)',
+                'inset-list-add': 'inset -1px 0 0 0 rgba(8, 8, 8, 0.1), inset 1px 0 0 0 rgba(8, 8, 8, 0.1)',
+                'inset-hover-weak': 'inset 0 0 0 1px rgba(8, 8, 8, 0.15)',
+                'inset-hover-strong': 'inset 0 0 0 1px rgb(45, 50, 50)',
+            },
```

Comment:
I've had a chat with Simone, and we both feel really strongly about this because aesthetically it looks much better with inset box-shadow than it does with borders that we have in Fondue.

Would it be fine for you that we wait for Jamie to come back from vacation and see his opinion regarding this?